requirements-ci.txt

```diff
@@ -0,0 +1,56 @@
+appdirs
```

Comment:
Here as discussed in the meeting, you should have only direct dependencies ```
```
pennylane
cirq-core
cirq-pasqal
numpy
```